test/test_torchinductor_opinfo.py

```diff
@@ -175,11 +170,9 @@ def process(device_type):
     "scatter_reduce.mean": {f16, f32, f64, i32, i64},  # segfault
     "scatter_reduce.prod": {f16, f32, f64, i32, i64},  # segfault
     "scatter_reduce.sum": {b8, f16, f32, f64, i32, i64},  # segfault
-    "softmax": {b8, f16, f32, f64, i32, i64},  # segfault
+    "softmax": {b8, f64, i32, i64},  # segfault
```

Comment:
what's happening for softmax and f16, f32?
can you track that issue somewhere if it isn't already

Reply:
fp16 and fp32 are passing, I removed them from skips. For integer inputs something wrong is going on with type promotion I think, for fp64 triton frequently has trouble generating code.